ShopCMS/views.py

```diff
@@ -65,7 +68,105 @@ class ProductCategoryViewSet(viewsets.ModelViewSet):
     def perform_create(self, serializer):
         serializer.save()
 
-
+    def subcategories(self, catagoryNumber):
+        category = ProductCategory.objects.get(id=catagoryNumber)
+        subcategories = category.get_all_subcategories()
+        return subcategories
+
+    def get_catagory_name(self, catagoryNumber):
+        return ProductCategory.objects.get(id=catagoryNumber).name
+
+
+
+    @action(detail=True, methods=['get'])
+    def get_sections(self, request, pk=None):
```

Comment:
Would like to see less hard coding, but is fine for now. Would possibly want to remove this function in the future and add more logic on the frontend to render the nav bar.